lib/tasks/validate.rake

```diff
@@ -0,0 +1,91 @@
+namespace :validate do
+  namespace :clusters do
+    task :all => Pathname.glob((Pathname.new(ENV['OOD_CLUSTERS'] || '/etc/ood/config/clusters.d').expand_path).join("*.yml")).map{|c| (c.to_s).to_sym }
+    
+    Pathname.glob((Pathname.new(ENV['OOD_CLUSTERS'] || '/etc/ood/config/clusters.d').expand_path).join("*.yml")).each do |config|
+        desc "Validate the cluster config: #{config}"
+        task (config.to_s).to_sym do
+            config_json = YAML.safe_load(config.read).to_json
+            puts "Validating cluster config: #{config}..."
+            if (JSON.parse(config_json)["v2"]["metadata"]["title"]).downcase.include?('quick')
+                schema = File.read((File.dirname(__FILE__) +'/quick_config_schema.json'))
+            else
+                schema = File.read((File.dirname(__FILE__) +'/config_schema.json'))
+            end
+            
+            if !JSON::Validator.validate(schema, config_json)
+              puts "Test for '#{config}' FAILED!"
+              puts JSON::Validator.fully_validate(schema, config_json)
```

Comment:
@ericfranz do we want this to fail on the first error, or run through all the configs and then either succeed or fail at the end?

Reply:
I would expect a validation for an individual cluster config to output a validation failed with the reasons, or validation succeeded, for each cluster.

Reply:
Then we'll want to add a failed flag that is set so that we can do something like `raise 'One or more cluster configurations are invalid.'` at the end.

With an explicit failed return code from the `raise` this could be tied into a CI system.